meilisearch/index.py

```diff
@@ -289,7 +289,7 @@ def add_documents(
         self,
         documents: List[Dict[str, Any]],
         primary_key: Optional[str] = None,
-    ) -> Dict[str, int]:
+    ) -> Dict[str, Any]:
```

Comment:
There is no type definition in PHP ?

Reply:
What do you mean? Something like:
`Dict[str, str|int]:`

Reply:
It receives a Task so it could be `int`, `str`, or even `Dict`.

Reply:
Something like 
```
Type: {

  uid: Int, 
  indexUid: string
}

```

etc

Reply:
There is a [TypedDict](https://docs.python.org/3/library/typing.html#typing.TypedDict), but it is first available in Python 3.8. You can do it all the way back to Python 3.6 buy using [Pydantic](https://pydantic-docs.helpmanual.io/) classes instead of dictionaries, but that would be a big change.

Reply:
@bidoubiwa, yes we could create a task `class` to use on the prototype on all these functions you can create an issue about it!
And yes, if we do, of course, we will be inspired by your models @sanders41 thank you very much for the reference.

Reply:
This could represent a breaking change for the user in any form?

Reply:
@brunoocasali it depends on how you look at it. Really `Dict[str, int]` has been wrong since tasks were introduced and I am thinking @alallema is first to notice and made the correction here. So really it was breaking in a previous release and a bug fix here.

The reason this was easy to miss is a dictionary is returned, but mypy has no way to know what the dictionary will be with the current implementation. Since all it knows you you said you are returning a dictionary and one is being returned it assumes you have the rest correct. The point @bidoubiwa brings up would prevent this from happening because at that point mypy would know exactly what to expect in the return values.

Reply:
I made an issue https://github.com/meilisearch/meilisearch-python/issues/449